frontend/app/API.js

```diff
@@ -1,63 +1,6 @@
 import { browserHistory } from 'react-router'
 import Raven from 'raven-js'
 
-const parse = {
```

Comment:
it was confusing to add a field to a model on the backend and then not be able to use it in a React component because it hadn't been whitelisted in parse